test/over_react/component_declaration/component_base_test.dart

```diff
@@ -946,7 +937,7 @@ dynamic getDartChildren(var renderedInstance) {
   assert(isDartComponent(renderedInstance));
   return getProps(renderedInstance)['children'];
 }
-
+var fooRef;
```

Comment:
Is this unused?

Reply:
its used on line 959 and set to the ref of a div.

Reply:
Maybe I'm not seeing it, but is there a purpose to setting this ref?

Reply:
Idk, but there was a string one before which is why i changed it to a callback ref lol ill try removing and seeing if tests fail.